modules/openapi-generator/src/main/resources/cpp-qt-client/helpers-header.mustache

```diff
@@ -264,6 +243,37 @@ bool fromJsonValue(QMap<QString, T> &val, const QJsonValue &jval) {
     return ok;
 }
 
+template <typename T>
+class OptionalParam {
+public:
+    T m_Value;
+    bool m_isNull = false;
+    bool m_hasValue;
+public:
+    OptionalParam(){
+        m_hasValue = false;
+    }
+    OptionalParam(const T &val, bool isNull = false){
+        m_hasValue = true;
+        m_Value = val;
+        m_isNull = isNull;
+    }
+    bool hasValue() const {
+        return m_hasValue;
+    }
+    T value() const{
+        return m_Value;
+    }
+
+    QString stringValue() const {
+        if (m_isNull) {
+            return QStringLiteral("");
+        } else {
+            return toStringValue(value());
+        }
+    }
+};
+
```

Comment:
Why did you moved this block? It makes review harder not seeing diff properly.

Reply:
It needs the toStringValue declarated and I thought it is easier to move after them.

Reply:
Ok I diffed the change and that is fine for me!